<ui version="4.0" >
 <class>XDMFPanel</class>
 <widget class="QWidget" name="XDMFPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>504</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Domain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DomainNames" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="pqTreeWidget" name="GridNames" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>13</hsizetype>
       <vsizetype>3</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Grids</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="pqTreeWidget" name="Variables" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>13</hsizetype>
       <vsizetype>3</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Variables</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Range(s)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Stride</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Stride:0" />
     </item>
     <item>
      <widget class="QSpinBox" name="Stride:1" />
     </item>
     <item>
      <widget class="QSpinBox" name="Stride:2" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="Parameters_label" >
     <property name="text" >
      <string>Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="Parameters" >
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>3</number>
     </property>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pqTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
